src/commands/auth/ctx.js

```diff
@@ -42,17 +42,19 @@ The following options exist for this command:
 
 Currently it is not possible to update the Adobe Adobe IMS context configuration
 using this command. Use the "aio config" commands for this.
-     e.g. aio config:set \\$ims.your_context.your_context_key "your_context_value"
+     e.g. aio config:set ims.your_context.your_context_key "your_context_value"
```

Comment:
sorry! missed this one. ims.contexts.your_context.your_context_key

Reply:
thanks for catching this!